package common.scalable;

message PrimaryPropRFData
{
    message Detail
    {
        required int32 cur_add_energy = 1;
        required int32 cur_level      = 2;
    }

    required int32 last_recover_time = 1;
    required int32 cur_energy        = 2;
    repeated Detail detail           = 3;
}

